src/Pages/3DPrintingConsole/requestForm.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './3D-console.css'
+import {
+  Button,
+  ButtonGroup,
+  Card,
+  CardTitle,
+  CardBody,
+  Collapse,
+  FormGroup,
+  Row,
+  Col
+} from 'reactstrap'
+
+function RequestForm (props) {
+  const { item } = props
+
+  const Console3DInfo = [
+    { lgSize: '3', title: 'E-mail/Contact:', value: item.projectContact },
+    { lgSize: '3', title: 'Requested Date:', value: item.date },
+    { lgSize: '2', title: 'Progress:', value: item.progress }
+  ]
+  const Console3DButtons = [
+    {
+      color: 'primary',
+      value: 'Reject',
+      handleUpdates: e => {
+        props.onHandleUpdateProgress(item, e)
```

Comment:
by changing the prop names to this component you must also do
```suggestion
        props.handleUpdateProgress(item, e)
```